docs/src/pages/fr/installation.md

```diff
@@ -85,13 +85,13 @@ Vous pouvez aussi remplacer la section "scripts" du fichier `package.json` avec
 ```diff
   "scripts": {
 -    "test": "echo \"Error: no test specified\" && exit 1"
-+    "start": "astro dev",
++    "dev": "astro dev",
 +    "build": "astro build"
   },
 }
 ```
 
-La commande [`start`](#start-astro) démarre le serveur de développement Astro à l'adresse `http://localhost:3000`. Une fois votre projet terminé, la commande [`build`](#build-astro) produit votre site dans le dossier `dist/`. [En savoir plus sur le déploiement d'un site développé avec Astro.](/guides/deploy)
+La commande [`dev`](#start-astro) démarre le serveur de développement Astro à l'adresse `http://localhost:3000`. Une fois votre projet terminé, la commande [`build`](#build-astro) produit votre site dans le dossier `dist/`. [En savoir plus sur le déploiement d'un site développé avec Astro.](/guides/deploy)
```

Comment:
hmmm shouldn't this link to #demarrere-astro not #start-astro? might be a PR to make sure the links are correct.

Reply:
Maybe it makes sense to manually set the id name for each heading? so that all the links are the same across languages?

Reply:
Oh, good catch, these may be broken in the translated page. I haven't been checking for these in reviews.

Reply:
I guess we should actually be using the link checking lint 😅

Reply:
oh yea, didn't we add that in the past? +1 for reenabling or adding a link checker!